### DISCUSSION TITLE
MY Github is hacked.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#129617] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i didn't use my github for a few days and today i saw some new unknown repos in my account and then i checked my login history and i got to know that i am hacked for the past 3 days hackers are creating repos of illegal mods of some games 
![image](https://github.com/community/community/assets/106374555/90058b82-270b-437e-9935-cdbe5b917224)
MY location and my o auth tokens have been revoked multiple times 
![image](https://github.com/community/community/assets/106374555/d9fd4d9f-abef-45ae-be33-20f70b7408ae)
The country change in my github happened multiple times to US , virginiaetc. but when i looked at the ip the locations are from india .
I don't know now what to do to save my account should i just backup my files and delete the whole account 
The hackers have a copyright in the repos named **catch me if you can**
![image](https://github.com/community/community/assets/106374555/134fc753-38fe-45c6-92ab-86059208d24a)
They are using my account to spread their mods i didn't even they are legit mods or some hackiing program so i don't installed it.
Pls help me what should I dooo.
Pls I need help!!!

**Comment by LiteBrite82** [comment]
Hi @akshatsharma1290 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

If you have 2FA enabled on your account, you may try the [account recovery method for 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)
